src/EventStore.Core/ClusterVNodeOptionsValidator.cs

```diff
@@ -1,6 +1,7 @@
 ﻿using System;
 using System.IO;
 using EventStore.Common.Exceptions;
+using EventStore.Core;
```

Comment:
Also not used.

Reply:
This is used. We need this because in Line 13, we make use of class ClusterVNodeOptions, for which, we need this.
![image](https://github.com/EventStore/EventStore/assets/88392797/ccaeaa9b-9f1f-4743-82fb-81c1b95e32ff)

Reply:
hmm weird, it's already in the `EventStore.Core` namespace 🤔

Reply:
Fixed in the latest commit.